src/frontend/components/material-admin/js/modules/form.js

```diff
@@ -12,7 +12,7 @@ materialAdmin
         return {
             restrict: 'C',
             link: function(scope, element) {
-                if($('.fg-line')[0]) {

```

Comment:
did you check the code in the old project to see if why this index [0] worked previously?

Reply:
@pbson `$('.fg-line')` will always exist I suppose (as a jQuery object). The real element that you want to find is `$('.fg-line')[0]`.

Reply:
@huy-ta For some reason `$('.fg-line')[0]` is undefined, but `$('.fg-line').first()` works, so I think I'm gonna change it to that. What's your opinion?

Reply:
`first()` will also always be a jQuery object, so it will always exists even if the element isn't found.

Reply:
For everybody: 
1/ the original code is very dirty. Basically it means "if there is at least one element on the whole page that has the `fg-line' class. I know this is material admin code and not ours, but still it's ugly.

2/ the condition $('.fg-line') will ALWAYS BE TRUE. Because what is returned will be a jQuery object. That jQuery object may contains 0 elements, still it's an object, so it will be evaluated to truthy.

Reply:
Can you try with `element.closest('.fg-line').length` ?

Reply:
OK I'll try it. Thank you. For some reason `$(.fg-line)[0]` does exist when using browser console, but return undefined when I try to console log it from the code

Reply:
@pbson: maybe it's "evaluated just now". You might recognize it by an exclamation mark next to the log. It means when the console.log statement runs, the value is undefined. But after that, the value is evaluated. But it's just a guess

Reply:
Maybe we can process differently. In Inbox, Click on "New folder": the new folder form is in a modal and uses fg-line, and it got the blue line working fine. Maybe you can spot the difference from here... ?